daemon/src/model/cfd.rs

```diff
@@ -502,6 +506,7 @@ impl Cfd {
             during_contract_setup: false,
             during_rollover: false,
             settlement_proposal: None,
+            funding_rate: FundingRate::new(Decimal::ZERO).expect("be valid"),
```

Comment:
❓ What happens when we calculate the funding fee with a fund rate of 0.0? Should we add a test?

Reply:
it just doesn't collect fees - I will add a test to make that obvious if it helps! :)